### PR TITLE
Feature: Add utils functions and unit tests to test S3 buckets

### PR DESCRIPTION
**Pull request by snova-francescar**
Add utils functions and unit tests to test S3 buckets.

### CODE REVIEW

File: utils/prod/s3_utils.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+import boto3
+from botocore.exceptions import ClientError
+
+# Setup logging
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+
+# Get the S3 bucket name
+BUCKET_AWS_NAME = os.getenv('BUCKET_AWS_NAME')
```
Comment (snova-jorgep) [inline utils/prod/s3_utils.py:19]: should we include a load_dotenv here, or we will always expect the envs to be set when when this is imported?

Comment (snova-jorgep) [reply utils/prod/s3_utils.py:19]: same comment for access keys in methods

Comment (snova-francescar) [reply utils/prod/s3_utils.py:19]: Good point.

Comment (snova-francescar) [reply utils/prod/s3_utils.py:19]: It is already there on line 3. I'll move it down for readability.

**Review by snova-luiss — APPROVED** [review]
lgtm

**Review by snova-kwasia — APPROVED** [review]
Approved - @snova-francescar . Thank you. Quick note - in a follow up PR. Let's add the unit test to the main test suite.

**Pull request merged**